common/schema/syntaxflow_rule.go

```diff
@@ -88,7 +88,7 @@ type SyntaxFlowRule struct {
 	TitleZh     string
 	Description string
 	Tag         string
-
+	AlertDesc   string
```

Comment:
just save Struct type, then can use this in alltime, and just marshal/unmarshal once for one instance.